_develop/connect/get-started.md

```diff
@@ -142,4 +142,24 @@ Then, you can use API methods under the `blockstackConnect` global variable:
 ```javascript
 const authOptions = { /** See docs above for options */ };
 blockstackConnect.showBlockstackConnect(authOptions);
+```
+
+## Handling redirect fallbacks
+
+Connect is built to use popups with the `window.postMessage` API, which provides a much better and seamless user experience. However, there are times when this flow can fail. For example, the popup may be blocked, or the `window.postMessage` API might not work properly (which often happens on mobile browsers).
+
+To make sure your app handles this gracefully, you'll need to handle the case where authentication is performed through vanilla redirects. With redirects, your users will be sent back to your app at a URL like:
```

Comment:
"vanilla redirects" is something I got hung up on a bit - is this for vanilla JS redirects?